src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.container {
+  display: flex;
+  padding: 40px;
+  align-items: center;
+  justify-content: center;
+}
+
+.calendar {
+  width: $calendar-width;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: $primary-gap;
+  
+  &__day {
+    width: 100px;
+    height: 100px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    border: 1px solid #000;
+    transition: all 0.5s ease-in-out;
+
+    &:hover {
+      transform: translate(0, -20px);
+      background-color: pink;
+      cursor: pointer;
+    }
+  }
+
+  @for $index from 1 through $mouth-days {
+    &__day:nth-child(#{$index}n):before {
```

Comment:
Use nesting, you can write this into `&__day`

Reply:
No, if i will cut from here, and copy into &__, letters, will not create, I thought above hour few days ago, why letters were not creating))